theme/views.py

```diff
@@ -80,15 +82,24 @@ def get_context_data(self, **kwargs):
                     pass
                 else:
                     # filter out any resources the requesting user doesn't have access
+                    num_private_resources = resources.filter(Q(raccess__public=False) |
+                            Q(raccess__discoverable=False)).count
                     resources = resources.filter(
                         Q(pk__in=self.request.user.uaccess.view_resources) |
                         Q(raccess__public=True) |
                         Q(raccess__discoverable=True))
             else:
                 # for anonymous requesting user show only resources that are either public or
                 # discoverable
+                
```

Comment:
Why is this assignment done twice?

Reply:
That was a mistake, thank you for the catch!